### ISSUE TITLE
Warning for undefined parameter `help`

### ISSUE THREAD

**Issue by mateuszpawlik** [nextflow-io/nf-validation#151]
With Nextflow 23.10.1 I get the following warning:
```
WARN: Access to undefined parameter `help` -- Initialise it to a default value eg. `params.help = some_value`
```
Where and how should I initialize `help` parameter?

**Comment by nvnieuwk** [comment]
Simply adding `params.help = some_value` to your `nextflow.config` file should do the trick :)

**Comment by mateuszpawlik** [comment]
This actually triggers the help message without executing the pipeline or an error.

Setting `params.help = "bla"` causes `ERROR ~ Specified param 'bla' does not exist in JSON schema.`

Setting `params.help = 42` cases printing the help message.

**Comment by mirpedrol** [comment]
It should be initialized as false `params.help = false`, otherwise it will be interpreted as true, and this show the help message.
You may also need to add this parameter to the nextflow schema to avoid the `does not exist in JSON schema` error.

**Comment by mateuszpawlik** [comment]
I added `params.help = false` separately to my other `params { ... }` The warning is gone and it didn't complain that it is not in the schema.
